src/parser/hunter/beastmastery/modules/spells/AspectOfTheWild.tsx

```diff
@@ -35,7 +35,7 @@ class AspectOfTheWild extends Analyzer {
     }
     if (this.casts === 0) {
       this.casts += 1;
-      this.spellUsable.beginCooldown(SPELLS.ASPECT_OF_THE_WILD.id, { timestamp: this.owner.fight.start_time });
+      this.spellUsable.beginCooldown(SPELLS.ASPECT_OF_THE_WILD.id, event);
```

Comment:
Not sure if this is something you want to update, but this line looks like its meant to resolve pre-pull casts of Aspect. By updating Buffs.ts and adding `triggeredBySpellId: SPELLS.ASPECT_OF_THE_WILD.id` to the declaration there, the PrepullNormalizer should handle all of that for you (and fabricate a cast event to fix cooldown usage trackers)

Reply:
Done in commit 85d830f